tests/conftest.py

```diff
@@ -0,0 +1,62 @@
+import asyncio
+
+import pytest
+from aioneo4j import Neo4j
+
+
+@pytest.fixture
+def loop(request):
+    with pytest.raises(RuntimeError):
+        asyncio.get_event_loop()
+
+    loop = asyncio.new_event_loop()
+
+    loop.set_debug(True)
+
+    request.addfinalizer(lambda: asyncio.set_event_loop(None))
+
+    try:
+        yield loop
+    finally:
+        loop.call_soon(loop.stop)
+        loop.run_forever()
+        loop.close()
+
+
+@pytest.fixture
+def neo4j(loop):
+    neo4j = Neo4j(loop=loop)
+    neo4j.auth = ('neo4j', 'neo4jneo4j')
+
+    try:
+        yield neo4j
+    finally:
```

Comment:
this is very bad idea to not to mock actual neo4j and drop everything from it.
if someone runs local neo4j and he runs tests he'll loose all data.